backend/src/common/enums/course/course-host.enum.ts

```diff
@@ -1,5 +1,6 @@
 enum CourseHost {
   W_UDEMY = 'www.udemy.com',
   UDEMY = 'udemy.com',
+  EDX = 'www.edx.org',
```

Comment:
```suggestion
  W_EDX = 'www.edx.org',
  EDX = 'edx.org',
```
Otherwise it won't work, if user passes edx.org/course/...